crates/uv-client/src/registry_client.rs

```diff
@@ -78,11 +79,15 @@ impl RegistryClientBuilder {
 
     pub fn build(self) -> RegistryClient {
         let client_raw = {
+            let timeout: u64 = env::var("UV_REQUEST_TIMEOUT")
+                .ok()
+                .and_then(|v| v.parse::<u64>().ok())
+                .unwrap_or(300);
```

Comment:
Can we log the timeout we're going to use e.g. with `debug!`?

Reply:
> Will we silently ignore invalid values here? Should we warn if it can't be parsed?
> Can we log the timeout we're going to use e.g. with `debug!`?

Thanks for the review.
Yes, I think it would be more user-friendly if you suggested it.
I have fixed the implementation and would appreciate a check.

Reply:
The following is how the implemented logs are generated.

```bash
root@484f49cfa3cf:~# /uv pip install torch -v 
 uv::requirements::from_source source=torch
    0.002100s DEBUG uv_interpreter::virtual_env Found a virtualenv named .venv at: /root/.venv
    0.002159s DEBUG uv_interpreter::interpreter Detecting markers for: /root/.venv/bin/python
    0.021205s DEBUG uv::commands::pip_install Using Python 3.10.12 environment at /root/.venv/bin/python
    0.021605s DEBUG uv_client::registry_client Pip request timeout is 300.
...
root@484f49cfa3cf:~# UV_REQUEST_TIMEOUT=a /uv pip install torch
Warning: UV_REQUEST_TIMEOUT is an invalid value. Using default value.
Resolved 22 packages in 891ms
████░░░░░░░░░░░░░░░░ [5/22] jinja2==3.1.3
```